inception-doc/src/main/resources/META-INF/asciidoc/admin-guide/installation_docker.adoc

```diff
@@ -100,7 +100,7 @@ annotations and knowledge bases, these are stored in files on disk). For product
 recommended to use a separate MySQL database instead of the embedded SQL database.
 
 You can do so by providing a second Docker for MySQL (see for example link:https://hub.docker.com/_/mysql/[this one]).
-We provide a docker-compose example file, which combines the two containers. In order to use this, download link:https://raw.githubusercontent.com/inception/inception/master/inception-app-webapp/src/main/docker/docker-compose.yml[docker-compose.yml] and place it into any folder, change to that folder, and issue the commands
+We provide a docker-compose example file, which combines the two containers. In order to use this, download link:https://raw.githubusercontent.com/inception-project/inception/master/inception-app-webapp/src/main/docker/docker-compose.yml[docker-compose.yml] and place it into any folder, change to that folder, and issue the commands
```

Comment:
Why are we linking to a URL on github and not to a docker-compose file included in the release JAR? I believe if this is turned into a relative link, it should be able to access the compose file from the JAR/Classpath through the usual static resources mechanism which also exposes the documentation HTML files.

Reply:
No special reason except time. I just did it like it was in Webanno and quick enough so it can be in 9.1

Reply:
I moved the file so it is shipped in the JAR and change the link in the documentation.